apps/barista-design-system/src/components/icon-color-wheel/icon-color-wheel.ts

```diff
@@ -62,11 +74,18 @@ export class BaIconColorWheel {
   _iconFillColor = DtColors.GRAY_700;
   /** @internal all colored circle blobs */
   _coloredBlobs: BaColorWheelBlob[];
+  /** @internal whether context is in browser */
+  isBrowser: boolean;
 
   /** whether the icon should be downloaded as png */
   private _convertToPng = false;
 
-  constructor(private _sanitizer: DomSanitizer) {
+  constructor(
+    private _sanitizer: DomSanitizer,
+    @Inject(PLATFORM_ID) private platformId: Object,
+    private _renderer2: Renderer2,
+  ) {
+    this.isBrowser = isPlatformBrowser(this.platformId);
```

Comment:
I guess the platform check is not necessary if you use the renderer to change the view.